test/UnitConvert.cpp

```diff
@@ -49,6 +49,7 @@ class UnitConvert : public UnitWSD
 
         config.setBool("ssl.enable", true);
         config.setInt("per_document.limit_load_secs", 30);
+        config.setBool("storage.filesystem[@allow]", false);
```

Comment:
This is to make sure that we test that convert-to still works when filesystem storage is disabled.